### DISCUSSION TITLE
是否有希望适配安卓 8.0.0 ?

### DISCUSSION THREAD

**Discussion by ellaweeks** [LSPosed/LSPosed#2173] [Q&A] [answered]
使用的时候最低要求是 oreoMR1 ,是否可以向下 适配 安卓 8.0.0 ?

**Comment by yujincheng08** [comment, marked as answer]
8.0 没有 [SharedMemory](https://developer.android.com/reference/android/os/SharedMemory)，我们依赖这个。

BTW，Java hook 框架 LSPlant 本身是支持到 5.0 的。

**Reply by yujincheng08** [reply]
补充一下，我们加载模块依赖 InMemoryDexClassloader，这个是只到 8.0 的。不过这个在 LSPlant 内部有个到 5.0 的兼容。
不管怎样技术上应该没啥问题，就是 **没有必要费大力气去为了极小部分人兼容老系统**。
